service/src/main/java/uk/gov/hmcts/reform/fpl/events/CallbackEvent.java

```diff
@@ -4,6 +4,7 @@
 
 public class CallbackEvent {
 
+    //TODO use RequestData here
```

Comment:
do we still need this todo?

Reply:
It's techdebt, maybe I should make it clearer in the comment 👍